android/src/main/java/com/github/reactNativeMPAndroidChart/charts/LineChartManager.java

```diff
@@ -55,8 +59,8 @@ void dataSetConfig(IDataSet<Entry> dataSet, ReadableMap config) {
         if (BridgeUtils.validate(config, ReadableType.Boolean, "drawCircles")) {
             lineDataSet.setDrawCircles(config.getBoolean("drawCircles"));
         }
-        if (BridgeUtils.validate(config, ReadableType.Boolean, "drawCubic")) {
-            lineDataSet.setDrawCubic(config.getBoolean("drawCubic"));
+        if (BridgeUtils.validate(config, ReadableType.String, "mode")) {
```

Comment:
you did not change `drawCubic` to `mode` prop type in some `lib/*Chart.js` files.